packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -52,37 +51,12 @@ const WizardHeading = ({ country_standpoint, currency, is_isle_of_man_residence,
         return <Localize i18n_default_text='Add a Deriv Gaming account' />;
     }
 
-    if (iom_signup && is_isle_of_man_residence) {
-        return <Localize i18n_default_text='Add a Deriv account' />;
-    }
-
     switch (real_account_signup_target) {
-        case 'malta':
-            if (
-                country_standpoint.is_united_kingdom ||
-                country_standpoint.is_rest_of_eu ||
-                country_standpoint.is_belgium
-            ) {
-                return <Localize i18n_default_text='Add a real Deriv Options account' />;
-            }
-            return <Localize i18n_default_text='Add a Derived account' />;
-        case 'iom':
-            if (country_standpoint.is_united_kingdom) {
-                return <Localize i18n_default_text='Add a real Deriv Gaming account' />;
-            }
-            return <Localize i18n_default_text='Add a Derived account' />;
         case 'maltainvest':
-            if (
-                country_standpoint.is_united_kingdom ||
-                country_standpoint.is_france ||
-                country_standpoint.is_other_eu ||
-                country_standpoint.is_rest_of_eu
-            ) {
+            if (country_standpoint.is_france || country_standpoint.is_other_eu || country_standpoint.is_rest_of_eu) {
```

Comment:
@matin-deriv Don't we offer deriv-app in UK anymore?

Reply:
@mahdiyeh-deriv Sure I will take care of that as well.
You're just a bit early to review 😅 I'm still working on the PR.

Reply:
We do not allow new signups and no more new real account signup.

<img width="469" alt="Screenshot 2024-04-04 at 11 28 37 AM" src="https://github.com/binary-com/deriv-app/assets/70938039/aa605cb2-cbcc-40bc-adb3-9fc0f0bbd609">